homework-3/src/views/SignIn.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { StylBoxMain } from '../components/common/Boxes/StylBoxMain';
+import FormSignIn from '../components/Forms/FormSignIn';
+import BoxFormSignIn from './../components/common/Boxes/StylBoxFormSignIn';
+import BoxContentLink from './../components/Boxes/BoxContentLink';
+import Loader from './../components/Loader/Loader';
+
+const SignUp = ({ title }) => {
+  React.useEffect(() => {
+    document.title = title || '';
+  }, [title]);
+
+  const [loading, setLoading] = React.useState(true);
+
+  React.useEffect(() => {
+    const loadData = async () => {
+      await new Promise((r) => setTimeout(r, 2000));
+
+      setLoading((loading) => !loading);
+    };
+
+    loadData();
+  }, []);
+
+  if (loading) {
```

Comment:
u can change if  block and set something like that 
`{loading ? <Loader /> :  all your components in else block }`
I think a simple ternary operator is better for understanding that if else block.

Reply:
Removed and moved to Router in component `Suspense`